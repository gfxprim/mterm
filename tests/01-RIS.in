2 2
\016j\ecj
