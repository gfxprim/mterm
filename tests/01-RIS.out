 --
|j |
|  |
 --
size 2x2 cursor 0x1
